CHANGELOG.md

```diff
@@ -411,6 +513,34 @@ Details
    https://github.com/owncloud/ocis/issues/3073
    https://github.com/cs3org/reva/pull/2977
 
+Changelog for reva 2.7.1 (2022-07-15)
```

Comment:
That's strange - the changelog generator decided to move the 2.7.1 section.

Reply:
we changed the template